setup.cfg

```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore = W503, E203, B305
+ignore = W503, E203, B305, PIE786
```

Comment:
I'm still opposed to ignoring this outright. Ignoring it per line is my preferred style, though if we really wanted to, we could add it to [per-file-ignores](https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-per-file-ignores):

```suggestion
ignore = W503, E203, B305
per-file-ignores =
    httpx/_client.py,httpx/_exceptions.py:PIE786
```

Reply:
I do agree with you. May be it's better to suppress the error per line back?

Reply:
That's my thought, but I'd like the other maintainers to chip in first.